packages/did-session/src/index.ts

```diff
@@ -321,6 +327,45 @@ export class DIDSession {
     return didWithCap
   }
 
+  /**
+   * Get a session for the given accountId, if one exists, otherwise creates a new one.
+   */
+  static async get(
+    account: AccountId,
+    authMethod: AuthMethod,
+    authOpts: AuthOpts = {}
+  ): Promise<DIDSession> {
+    if (typeof window === 'undefined')
+      throw new Error(
+        'DIDSession.get function requires browser environment, try using authorize instead'
+      )
+    if (!authOpts.resources || authOpts.resources.length === 0)
+      throw new Error('Required: resource argument option when authorizing')
+
+    const store = await SessionStore.create()
```

Comment:
Is there any risk of concurrency issue with the store getting created multiple times? Maybe the store could be a singleton?

Reply:
Hm, not sure. I think this might be handled by indexedDB?

Reply:
According to chatGPT, I believe we should be good assuming that two different instances of the IDBDatabase use the same lock under the hood.

#### Transactions

IndexedDB transactions can be used to help avoid race conditions. When you open a transaction, you can specify its mode as either `readonly` or `readwrite`. If a transaction is opened in `readwrite` mode, it obtains a write lock on the object store, ensuring that other `readwrite` transactions on the same object store are serialized. This can help prevent race conditions when multiple instances are writing to the database. However, read operations can still go through in parallel.

Reply:
Nice, thanks for confirming!